### PR TITLE
OM-2318 sensors fix

### PR DESCRIPTION
**Pull request by svanscho**
Return sensor object in set_sensor_configuration(s) API calls

### CODE REVIEW

File: testing/unittests/gateway_tests/webservice_test.py [hunk 1]
```diff
@@ -175,15 +176,29 @@ def test_sensor_configurations(self):
                 'virtual': False,
             }], json.loads(response)['config'])
 
+    def test_set_sensor_configuration(self):
```
Comment (LnL7) [inline testing/unittests/gateway_tests/webservice_test.py:179]: Is there also a test for the registration case? (post without id and response with)

**Review by LnL7 — APPROVED** [review]

File: testing/unittests/gateway_tests/webservice_test.py (hunk 1 above)
Comment (svanscho) [reply testing/unittests/gateway_tests/webservice_test.py:179]: There is a test in the sensor controller:
https://github.com/openmotics/gateway/blob/60f480b2d12a08dce7451e16be97225797a5f707/testing/unittests/gateway_tests/sensor_controller_test.py#L248
From an E2E perspective, would rather not test in the webservice as it requires some refactoring with not mocking the sensor controller any longer etc.

Comment (LnL7) [reply testing/unittests/gateway_tests/webservice_test.py:179]: these are not webservice tests but controller tests

Comment (LnL7) [reply testing/unittests/gateway_tests/webservice_test.py:179]: checking the id of the return dto in the return value here would cover it https://github.com/openmotics/gateway/blob/60f480b2d12a08dce7451e16be97225797a5f707/testing/unittests/gateway_tests/sensor_controller_test.py#L255.

**Pull request merged**